CONTRIBUTING.md

```diff
@@ -79,19 +79,21 @@ conda install sphinx
 Or, if using ``pip``, you can grab it with
 
 ```bash
+python -m pip install git+https://github.com/spyder-ide/spyder-docs-sphinx-theme.git
 pip install sphinx
 ```
 
-Now, ``cd`` into the ``doc`` directory in your clone of this repo, and run
```

Comment:
Why was this removed? The code block should have at least some description, e.g.

```markdown
Now, run the following on macOS/Linux:
```

Reply:
Great!